pom.xml

```diff
@@ -124,6 +124,7 @@
     <!--    default properties-->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <snappy.version>1.1.2.6</snappy.version>
+    <spark.binary.version>2.3</spark.binary.version>
```

Comment:
default version is not 2.4?

Reply:
No, because default profile is 2.3,  i have removed the global property and let the profiles decide the version